glam/settings.py

```diff
@@ -120,6 +120,8 @@ class Core(Configuration):
         ],
     }
 
+    GA_TRACKING_ID = os.getenv("GA_TRACKING_ID", "")
```

Comment:
Use the django-configurations `values` that is already imported for environment variables, please. 

https://django-configurations.readthedocs.io/en/stable/values/#value-class